### PR TITLE
style lints: lesson 3

### PR DESCRIPTION
**Pull request by wolovim**
## What does it do?

Quick editing round of Lesson 3 + OpenZeppelin Fundamental for grammar & punctuation consistency, and proper naming.

### CODE REVIEW

File: src/utils/questions/lesson-3/1-intro/Q5.json [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
     {
-      "answer": "We need a `.json` file to store our NFT data."
+      "answer": "NFT data must be stored in a `.json` file."
     },
     {
-      "answer": "We can store NFT data in a smart contract.",
+      "answer": "NFT data can be stored in a smart contract.",
       "correct": true
     }
   ]
```
Comment (elPiablo) [inline src/utils/questions/lesson-3/1-intro/Q5.json:11]: really appreciate all the re-wording on this question set - I remember really struggling with it - maybe it was a late-nighter!! 🤣

File: src/pages/lessons/projects/3.mdx [hunk 2]
```diff
@@ -91,14 +91,14 @@ tiered NFTs and uncover the unique superpowers they possess. By the end of this
...
-- Setting up the development environment
-- Writing and deploying a Solidity smart contract
-- Creating tiered NFTs with varying levels of rarity and attributes
-- Write scripts for deployment
-- Exploring additional functionalities and possibilities with tiered NFTs ... in
+- Set up a development environment
+- Write and deploy a Solidity smart contract
+- Create tiered NFTs with varying levels of rarity and attributes
+- Write a script for deployment
+- Explore additional functionalities and possibilities with tiered NFTs... in
```
Comment (elPiablo) [inline src/pages/lessons/projects/3.mdx:98]: I guess it's personal taste with the verb form. 

In my opinion, from a motivational perspective the continuous form is a nice touch for beginners here in the lead-in,  who might encounter some or all of these tasks as adventures in themselves, especially if they run into trouble i.e. create a bit of a safe space. 

We had made an explicit effort to actually rename the sub-sections in the way you're suggesting here (the conversation back then was to get learners used to open source conventions such as succinctly naming Github commits, etc). Is that the reason for homogenising them i.e. with the sub-headers in the lesson? 

I can go with it, but thought I'd share my rationale.

**Review by elPiablo — COMMENTED** [review]
Please see the comments on line 98. Something to bear in mind for the learning environment. Otherwise good to go.

File: src/pages/lessons/projects/3.mdx [hunk 3]
```diff
@@ -91,14 +91,14 @@ tiered NFTs and uncover the unique superpowers they possess. By the end of this
...
 
-- Setting up the development environment
-- Writing and deploying a Solidity smart contract
-- Creating tiered NFTs with varying levels of rarity and attributes
-- Write scripts for deployment
-- Exploring additional functionalities and possibilities with tiered NFTs ... in
+- Set up a development environment
+- Write and deploy a Solidity smart contract
+- Create tiered NFTs with varying levels of rarity and attributes
+- Write a script for deployment
```
Comment (elPiablo) [inline src/pages/lessons/projects/3.mdx:97]: 2 scripts: deploy.js and mint.js

File: src/pages/lessons/projects/3.mdx (hunk 2 above)
Comment (wolovim) [reply src/pages/lessons/projects/3.mdx:98]: only motivation here was consistent use of language. in this case, the odd one out was "Write scripts...", which could be converted to "Writing scripts..."

**Review by elPiablo — APPROVED** [review]
nice work. good to merge

**Pull request merged**